flaml/data.py

```diff
@@ -23,9 +23,11 @@
     SEQCLASSIFICATION,
     MULTICHOICECLASSIFICATION,
     TOKENCLASSIFICATION,
+    "mm_multi",
```

Comment:
can you automatically detect "mm_multi" and "mm_binary" so we don't need these two values anymore?